### DISCUSSION TITLE
编译问题，这个是什么问题呢，windows环境

### DISCUSSION THREAD

**Discussion by fulianchen** [shadowsocks/shadowsocks-android#2830] [Q&A]
23:16	Gradle sync started

23:16	Gradle sync failed: Unsupported method: ProductFlavor.getSigningConfig().
			The version of Gradle you connect to does not support that method.
			To resolve the problem you can change/upgrade the target version of Gradle you connect to.
			Alternatively, you can ignore this exception and read other information from the model.
			Consult IDE log for more details (Help | Show Log) (5 s 830 ms)
v

**Comment by cloud0435** [comment]
请问这个问题有解决么
